_topic_maps/_topic_map.yml

```diff
@@ -3264,12 +3264,15 @@ Topics:
   Topics:
   - Name: Network Observability Operator release notes 1.10
     File: network-observability-operator-release-notes-1-10
-  #- Name: Network observability release notes 1.9.3
-  #  File: network-observability-release-notes-1-9-3
-  #- Name: Network observability release notes 1.9.2
-  #  File: network-observability-release-notes-1-9-2
+#Keeping network-observability-operator-release-notes until all entries have been modularized.
+#Once all entire have been modularized, it will be updated with modules from 1-10 and these comments will be removed.
   - Name: Network observability release notes
     File: network-observability-operator-release-notes
+  - Name: Network Observability Operator archived release notes
+    Dir: release_notes_archive
+    Topics:
+    - Name: Network Observability Operator release-notes-archive
```

Comment:
Are you sure this is what you want the name to appear as? release-notes-archive?